modules/rosa-node-drain-grace-period.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+//this module applies to ROSA HCP only
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-node-drain-grace-period_{context}"]
+= Configuring node drain grace periods in {hcp-title} clusters
+
+You can configure the node drain grace period for machine pools in your cluster. The node drain grace period for a machine pool is how long the cluster respects the Pod Disruption Budget protected workloads when upgrading or replacing the machine pool. After this grace period, all remaining workloads will be forcibly evicted. The value range for the node drain grace period is from 0 to 1 week. With the default value 0, or empty value, the machine pool drains without any time limitation until complete.
+
+
+.Prerequisites
+
+* You installed and configured the latest {product-title} (ROSA) CLI, `rosa`, on your workstation.
+* You logged in to your Red Hat account by using the ROSA CLI.
```

Comment:
IBMSG suggests not to include logging in as part of a prerequisite step; https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#prerequisite-information. In this case, i think the installation and configuration of ROSA CLI should be a sufficient indicator that you need to be properly logged in to use ROSA.